scene/gui/text_edit.h

```diff
@@ -636,6 +638,15 @@ class TextEdit : public Control {
 	void set_right_click_moves_caret(bool p_enable);
 	bool is_right_click_moving_caret() const;
 
+	Selection::Mode get_selection_mode() { return selection.selecting_mode; };
+	void set_selection_mode(Selection::Mode p_mode, int p_line = -1, int p_column = -1);
+
+	int get_selection_line() { return selection.selecting_line; };
+	void set_selection_line(int p_line) { selection.selecting_line = p_line; };
```

Comment:
I can't see a use case for wanting to set `selection_line` or `selection_column` separately, without the mode, as `TextEdit` also changes these values, so would suggest removing these setters?